### PR TITLE
[ Switch] RTL feature

### PR DESCRIPTION
**Pull request by guanpu**
It looks changes many, but if you sync feat/rtl2 with master firstly and then review this MR, it would be ok as I rebased my feature branch with master.

### CODE REVIEW

File: src/switch/scss/variable.scss [hunk 1]
```diff
@@ -191,3 +191,6 @@ $switch-hover-off-border-color: $color-line1-3 !default;
 /// @level off
 /// @namespace statement/normal/bounding
 $switch-off-shadow: $shadow-1 !default;
+
+/// rtl shadow
+$switch-rtl-on-shadow: $shadow-1-left !default;
```
Comment (youluna) [inline src/switch/scss/variable.scss]: 这么写配置平台的shadow配置在rtl情况下就不生效了

**Comment by youluna** [comment]
box-shadow这个有什么好办法么 :(  @tao1991123 @guanpu

File: src/switch/scss/variable.scss (hunk 1 above)
Comment (guanpu) [reply src/switch/scss/variable.scss]: 加个注释让switch-rtl-on-shadow本身也可配置？

Comment (youluna) [reply src/switch/scss/variable.scss]: hold吧 先着么改着

**Pull request merged**